Loan.ts

```diff
@@ -220,7 +220,7 @@ export interface LoanRetrieveInterface {
 }
 export interface LoanDisbursementDetails {
   id: number,
-  expectedDisbursementDate: Array<number>,
+  expectedDisbursementDate: string,
```

Comment:
not required to change the earlier types, if these apis require them to be Array<number> we will put their type as Array<number> if they require to be string , then we will put them as string